src/gui/advancedsettings.cpp

```diff
@@ -323,6 +326,11 @@ void AdvancedSettings::loadAdvancedSettings()
     spinBoxAsyncIOThreads.setValue(session->asyncIOThreads());
     addRow(ASYNC_IO_THREADS, tr("Asynchronous I/O threads"), &spinBoxAsyncIOThreads);
 #endif
+    // Checking Memory Usage
+    spinBoxCheckingMemUsage.setMinimum(1);
+    spinBoxCheckingMemUsage.setMaximum(256000);
+    spinBoxCheckingMemUsage.setValue(session->checkingMemUsage());
+    addRow(CHECKING_MEM_USAGE, tr("Checking memory usage"), &spinBoxCheckingMemUsage);
```

Comment:
I think the name a bit too vague, maybe "Outstanding memory blocks when checking torrents".

Reply:
Agreed, but wouldn't it also be useful to also mention that each block is 16 KiB? Example:
"Outstanding 16 KiB memory blocks when checking torrents"
or
"Outstanding memory when checking torrents (16 KiB blocks)"
or
"Outstanding memory (16 KiB blocks) when checking torrents"

Or is it too much information/confusing?

Reply:
Ok, I will make that change and set the text to "Outstanding memory when checking torrents"